src/main/java/it/gov/pagopa/pu/classification/repository/AssessmentsRegistryRepository.java

```diff
@@ -98,4 +99,24 @@ void insertIfNotExists(
     @Transactional
     @Query("UPDATE AssessmentsRegistry a SET a.status = :status WHERE a.organizationId = :organizationId and a.debtPositionTypeOrgCode = :debtPositionTypeOrgCode and a.operatingYear = :operatingYear")
     void updateStatus(AssessmentsRegistryStatus status, Long organizationId, String debtPositionTypeOrgCode, String operatingYear);
+
+
+    @Query("""
+        SELECT entity
+        FROM AssessmentsRegistry entity
+        WHERE entity.organizationId = :organizationId
+            AND entity.debtPositionTypeOrgCode = :debtPositionTypeOrgCode
+            AND entity.sectionCode = :sectionCode
+            AND entity.officeCode = :officeCode
+            AND entity.assessmentCode = :assessmentCode
+            AND entity.operatingYear = :operatingYear
+    """)
+    Optional<AssessmentsRegistry> findByOrganizationIdAndCodes(
```

Comment:
This could return more than 1 entity, you should take, if exists, the record having ACTIVE status, or any record if there is not ACTIVE
```suggestion
    List<AssessmentsRegistry> findByOrganizationIdAndCodes(
```